tests/nomp-api-105.c

```diff
@@ -7,14 +7,14 @@
 // Calling nomp_jit() with valid functions should not return an error.
 static int test_valid_clauses(void) {
   int err = 0;
-  TEST_BUILTIN_TYPES(105_valid);
+  TEST_BUILTIN_TYPES(105_valid, );
   return err;
 }
 
 // Calling nomp_jit() with invalid functions should return an error.
 static int test_invalid_clauses(void) {
   int err = 0;
-  TEST_BUILTIN_TYPES(105_invalid);
+  TEST_BUILTIN_TYPES(105_invalid, );
```

Comment:
I don't think these `,` s are required now.

Reply:
`TEST_BUILTIN_TYPES` macro is not changed to `TEST_BUILTIN_TYPES(...)`.